bin/git-release

```diff
@@ -101,8 +103,17 @@ if test $# -gt 0; then
     git commit -a -m "$msg" --allow-empty
   fi
 
+  declare -a sign_args
```

Comment:
Since either `-s` or `-u` is allowed, maybe we should add an argument check to warn users when both `-s` and `-u` are provided?